src/bundle/Resources/public/js/scripts/helpers/language.helper.js

```diff
@@ -0,0 +1,5 @@
+const getLanguageCode = () => {
+    return window.document.querySelector('meta[name="LanguageCode"]')?.content;
```

Comment:
Should we consider using `ibexa.adminUiConfig.backOfficeLanguage` as a fallback, similarly to the  `getId` function in the user helper?

Reply:
This probably should be `content` helper.